internal/util/dir.go

```diff
@@ -31,6 +32,10 @@ func MkDirTemp(prefix string) (string, error) {
 	tempDirMutex.Lock()
 	defer tempDirMutex.Unlock()
 
+	if tempDir != "" {
+		MkDir(tempDir)
+	}
+
```

Comment:
Is this necessary?

Reply:
Unfortunately, we need to use MkDIr here because when we pass tempDir to os.MkdirTemp(tempDir, prefix), tempDir must already exist.